.github/workflows/backend.yml

```diff
@@ -0,0 +1,82 @@
+name: gradio-backend
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+concurrency:
+  group: backend-${{ github.ref }}-${{ github.event_name == 'push' || github.event.inputs.fire != null }}
+  cancel-in-progress: true
```

Comment:
Just to confirm, what this does is cancel the existing GH action that is in progress if someone pushes again to the same branch? We didn't do this for Circle CI so that it would be easier to trace a failure to a specific commit. I think we should keep that behavior

Reply:
Sounds good! Will set to False.